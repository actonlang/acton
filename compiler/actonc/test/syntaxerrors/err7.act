f : [a (Eq)] => (a,a) -> bool
