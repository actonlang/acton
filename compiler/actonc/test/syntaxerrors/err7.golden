Building file test/syntaxerrors/err7.act using temporary scratch directory

ERROR: Error when compiling err7 module: Syntax error

test/syntaxerrors/err7.act:1:6:
  |
1 | f : [a (Eq)] => (a,a) -> bool
  |      ^
unexpected 'a'
expecting type variable (upper case letter optionally followed by digits)

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 

